components/pagination/theme.css

```diff
@@ -0,0 +1,54 @@
+@import '@teamleader/ui-colors';
+@import '@teamleader/ui-typography';
+@import '@teamleader/ui-utilities';
+
+:root {
+  --ellipsis-min-width: calc(3 * var(--unit));
+}
+
+.pagination {
+  &:not(.inverse) {
```

Comment:
this `:not` is irrelevant since you're just overwriting all the properties in the `.inverse` scope, you can just remove the `:not` and everything will still work as before.

Reply:
@InstaK, I feel you, but here's the difference:

**With :not**
![schermafdruk 2017-12-20 09 02 56](https://user-images.githubusercontent.com/5336831/34196993-d1a5785a-e564-11e7-9fe6-7357be2e71c4.png)

**Without :not**
![schermafdruk 2017-12-20 08 56 14](https://user-images.githubusercontent.com/5336831/34197005-e0188e72-e564-11e7-859a-26a8c9d5ee98.png)

Why would we prefer to apply css that is overridden anyway?

Reply:
Fair enough, thanks for clarifying!

 :ok: by @InstaK on 20/12/2017 10:25:09